src/utils/numberFormat.js

```diff
@@ -1,8 +1,8 @@
 import numbro from 'numbro'
+import _ from 'lodash'
 
-export const toUSD = value => {
-  return numbro(value).formatCurrency({
-    thousandSeparated: true,
-    mantissa: 0
+export const toUSD = (value, mantissa) => {
+  return numbro(_.round(value, mantissa)).formatCurrency({
```

Comment:
Related to #41 . Now the 30th year in the amortization table finishes with 0 as opposed to -5, and numbers in the summary make sense as well.